app/views/configure/queries.phtml

```diff
@@ -61,9 +61,7 @@
 	</form>
 
 	<?php $class = $this->displaySlider ? ' class="active"' : ''; ?>
-	<a href="#" id="close-slider"<?= $class ?>>
-		<?= _i('close') ?>
-	</a>
+	<a href="#" id="close-slider"<?= $class ?>>❌</a>
```

Comment:
Was there a problem with the `_i('close')` approach?

Reply:
@Frenzie wrote: "This and the whole X strikes me as odd"

And now it is similar to https://github.com/FreshRSS/FreshRSS/blob/edge/app/layout/aside_feed.phtml#L118